.test_durations

```diff
@@ -0,0 +1,1198 @@
+{
```

Comment:
Are these files expected to be committed or should they be `gitignore`d?

Reply:
These do get committed, so that the tests have an estimate of how long the tests will take.